docs/readme/monitoring.md

```diff
@@ -1,16 +1,55 @@
 # Monitoring
-Metrics in Data Prepper are instrumented using [Micrometer.io](https://micrometer.io/).
+Metrics in Data Prepper are instrumented using [Micrometer.io](https://micrometer.io/). There are two types of metrics: 
+(1) JVM and system metrics; (2) Plugin metrics. Prometheus is used as the metrics backend.
```

Comment:
Why are we using Prometheus as the backend? The user should be able to choose.

Reply:
So far there is only prometheus. @kowshikn @wrijeff Any plan on supporting more options?